popupshop-form.php

```diff
@@ -0,0 +1,31 @@
+<?php get_header(); ?>
+
+<form>
+  <div class="popupshop-name">
+	  <p>Business / Project Name:<br /></p>
+	  <input type="text" name="name">
+	  <br />
+	  <p>Specialize in:<br /></p>
+	  <input type="text" name="specialize-in">
+	  <br>
```

Comment:
Same comments as previous PR.

Also not sure where you're going with these files but maybe will become clear later on. :)